### PR TITLE
Enable local repositories if needed

### PR DESCRIPTION
**Pull request by JRivrain**
- Related ticket: https://progress.opensuse.org/issues/47459
- Verification run: 
http://amazing.suse.cz/tests/3604
http://amazing.suse.cz/tests/3605

### CODE REVIEW

File: tests/autoyast/repos.pm [hunk 1]
```diff
@@ -25,6 +25,8 @@ sub run {
     # sles12_minimal.xml profile does not install "ip"
     assert_script_run 'ip a || ifstatus all';
     pkcon_quit;
+    # Enable local repositories if needed
+    zypper_call 'mr -el' # Returns 0 even if repos are already enabled
```
Comment (rwx788) [inline tests/autoyast/repos.pm]: You've removed import, which we need.

Comment (JRivrain) [reply tests/autoyast/repos.pm]: > You've removed import

If by import you mean "use version_utils qw(is_sle is_upgrade);" that I had added in previous PR, we do not need it it here anymore.

Comment (JRivrain) [reply tests/autoyast/repos.pm]: In case the repos are already enabled, it does just this http://amazing.suse.cz/tests/3605#step/repos/7

Comment (rwx788) [reply tests/autoyast/repos.pm]: Ah, utils is already imported. I thought `zypper_call` is in some other lib. Sorry for the confusion.

**Pull request merged**